src/components/renderer/SvgRenderer.tsx

```diff
@@ -0,0 +1,182 @@
+import { v2, utils } from "@govtechsg/open-attestation";
+import React, { CSSProperties, FunctionComponent, useEffect, useState } from "react";
+import { renderToStaticMarkup } from "react-dom/server";
+import crypto from "crypto";
+import bs58 from "bs58";
+import { ConnectionFailureTemplate, NoTemplate, TamperedSvgTemplate } from "../../DefaultTemplate";
+/* eslint-disable-next-line @typescript-eslint/no-var-requires */
+const handlebars = require("handlebars");
+
+interface SvgRendererProps {
+  document: any; // TODO: Update to OpenAttestationDocument
+  svgRef: React.RefObject<HTMLIFrameElement>;
```

Comment:
instead of forcing the a ref to be passed down by the user and using a `svgRef`, maybe we should stick to conventions by enabling the optional `ref` prop. This can be done via:
1. `React.forwardRef`
2. declaring a ref within the component and "merge" with the user's `ref` (if any) so that we can still `updateIframeHeight` independent of whether user passes us the `ref`


```
export const SvgRenderer = React.forwardRef<HTMLIFrameElement, SvgRendererProps>(function SvgRenderer(
  { document, svgData, style, className, sandbox = "allow-same-origin", onConnected, forceV2 = false },
  ref
) {
  const innerRef = useRef<HTMLIFrameElement>(null);
   // "merge" the refs, there is another way to do it, but this is more concise and shd work the same
  useImperativeHandle(ref, () => innerRef.current as HTMLIFrameElement);
  ...
        <iframe
          ref={innerRef}
  ...  
  
```